jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/test/java/org/jbpm/workbench/es/client/editors/jobdetails/JobDetailsPresenterTest.java

```diff
@@ -0,0 +1,140 @@
+        presenter.setExecutorService(callerMockExecutorService);
+    }
+
+    @Test
+    public void testOnSelectionEvent() {
+        String selectedServerTemplate = "serverTemplate";
+        Long jobId = 1L;
+        String deploymentId = "evaluation.1.0.1";
+        String key = "key";
+        RequestSummary job = new RequestSummary(jobId,
+                                                new Date(),
+                                                RequestStatus.QUEUED,
+                                                "commandName",
+                                                "Message",
+                                                key,
+                                                1,
+                                                0,
+                                                "processName",
+                                                1L,
+                                                "processInstanceDescription",
+                                                deploymentId);
+
+        RequestDetails requestDetails = new RequestDetails(job,
+                                                           null,
+                                                           null);
+        JobSelectedEvent event = new JobSelectedEvent(selectedServerTemplate,
+                                                      job.getDeploymentId(),
+                                                      job.getJobId());
+
+        when(placeManager.getStatus(any(DefaultPlaceRequest.class))).thenReturn(PlaceStatus.CLOSE);
+        when(executorServiceMock.getRequestDetails(selectedServerTemplate,
+                                                   deploymentId,
+                                                   jobId)).thenReturn(requestDetails);
+        presenter.onJobSelectedEvent(event);
+
+        verify(executorServiceMock).getRequestDetails(selectedServerTemplate,
+                                                      deploymentId,
+                                                      jobId);
+        verify(view).setValue(requestDetails);
+
+        ArgumentCaptor<ChangeTitleWidgetEvent> captor = ArgumentCaptor.forClass(ChangeTitleWidgetEvent.class);
+        verify(changeTitleWidgetEventMock).fire(captor.capture());
+        assertEquals(job.getId() + " - " + job.getKey() + " - " + job.getDeploymentId(),
+                     captor.getValue().getTitle());
+        assertEquals(jobId + " - " + key + " - " + deploymentId,
+                     captor.getValue().getTitle());
+    }
+
+    @Test
+    public void getJobDetailTitleTest() {
+        Long jobId = 1L;
+        String key = "key";
+        String deploymentId = "evaluation.1.0.1";
+        RequestSummary job = new RequestSummary();
+        job.setId(jobId);
+        job.setKey(key);
+        job.setDeploymentId(deploymentId);
+
+        assertEquals(job.getId() + " - " + job.getKey() + " - " + job.getDeploymentId(),
```

Comment:
I think for these expected values, it would be more interesting to have the final full String there, even for visualizing the result I guess it makes sense.